<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>640</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>640</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>640</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>covc</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>601</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetMaximumSize</enum>
     </property>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSave">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSave_metafile"/>
    <addaction name="actionLoad_metafile"/>
    <addaction name="actionAdd_image"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuCalculate">
    <property name="title">
     <string>Colorer</string>
    </property>
    <addaction name="actionRun"/>
   </widget>
   <addaction name="menuSave"/>
   <addaction name="menuCalculate"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionSave_metafile">
   <property name="text">
    <string>Save metafile</string>
   </property>
   <property name="toolTip">
    <string>Save</string>
   </property>
  </action>
  <action name="actionLoad_metafile">
   <property name="text">
    <string>Load metafile</string>
   </property>
  </action>
  <action name="actionAdd_image">
   <property name="text">
    <string>Add image</string>
   </property>
  </action>
  <action name="actionRun">
   <property name="text">
    <string>Run</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>319</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
